client/src/components/UI/__tests__/navBar.test.js

```diff
@@ -1,30 +1,55 @@
-/* eslint-disable import/no-extraneous-dependencies */
-/* eslint-disable no-unused-vars */
 import React from 'react';
-import { shallow } from 'enzyme';
+import { createShallow } from '@material-ui/core/test-utils';
+import { Button } from '@material-ui/core';
 import NavBar from '../NavBar';
 
+const createProps = ({
+  userContext = {}, classes = {},
+}) => ({
+  userContext, classes,
+});
+
+let wrapper; let props; let shallow;
+
+beforeEach(() => {
+  props = createProps({
+    userContext: {}, classes: {},
```

Comment:
`userContext` and `classes` will be empty object by default. No need to pass `{}` explicitly.

Reply:
changed it